packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -33,6 +34,13 @@ export interface ControlBarButtonStrings {
   offLabel?: string;
 }
 
+/**
+ * Styles for all {@link ControlBarButton} implementations.
+ *
+ * @public
+ */
+export interface ControlBarButtonStyles extends IButtonStyles {}
```

Comment:
is this for later on when (if) specific styles will be added?

Reply:
In the meantime, you could defined it like following to handle this issue:
export type ControlBarButtonStyles = IButtonStyles;

Reply:
Lint allows empty interfaces, and this pattern seems to be common in calling SDK.

The intention, that we are extending the interface is made more explicit that way.

Also, changing `type` to `interface` later would be a breaking change, I think.